lemur/api_keys/service.py

```diff
@@ -34,8 +36,9 @@ def revoke(aid):
     :return:
     """
     api_key = get(aid)
-    setattr(api_key, "revoked", False)
+    setattr(api_key, "revoked", True)
```

Comment:
Happened to observe this while working on log changes. This method is used only via CLI.
The revoke via lemur UI calls update method instead, so we may have never seen issue with revoke.